training_pipeline/pipeline/local_pipeline.py

```diff
@@ -50,6 +45,17 @@ def create_pipeline(
     )
     components.append(trainer)
 
+    pusher_args = {
+        "model": trainer.outputs["model"],
+        "push_destination": tfx.proto.PushDestination(
+            filesystem=tfx.proto.PushDestination.Filesystem(
+                base_directory=serving_model_dir
+            )
+        ),
+    }
+    pusher = Pusher(**pusher_args)  # pylint: disable=unused-variable
+    components.append(pusher)
+
```

Comment:
What is the difference between `local_pipeline.py` and `pipeline.py`?

Reply:
They are the almost same, but I just kept them in different files since some components and features are not available for local runner(i.e. RuntimeParameter). Think `local_pipeline.py` is there to check validity of the pipeline. 

Also I just want readers(who is afraid of using cloud technologies) to note that TFX pipeline can be run on local environment w/o GCP involved. 

But let's consider in the future PR to merge these two files.

Reply:
Sure. Have we added a note to differentiate between these scripts to not confuse the readers?

Reply:
yup, I wrote a section how to run local and vertex pipeline. let me write how they are different in the following-up PR to update README.